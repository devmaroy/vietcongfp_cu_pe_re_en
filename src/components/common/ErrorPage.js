import React from 'react';
import { Link } from 'react-router-dom';

const ErrorPage = () => (
    <div>
        <h1>Aaaah! Something went wrong.</h1>
        <p>Maybe the page, item you were looking for doesn't exist or something else is broken.</p>
        <p>Sorry :(</p>
        <Link to="/">Go home</Link>
    </div>
);

export default ErrorPage;