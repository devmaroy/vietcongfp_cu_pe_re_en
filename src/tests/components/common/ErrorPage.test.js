import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from '../../../components/common/ErrorPage';

test( 'should correctly render ErrorPage', () => {
    const wrapper = shallow( <ErrorPage /> );

    expect( wrapper ).toMatchSnapshot();
});