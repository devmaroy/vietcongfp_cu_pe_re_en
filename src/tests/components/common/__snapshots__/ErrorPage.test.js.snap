// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ErrorPage 1`] = `
<div
  className="content-container"
>
  <div
    className="page"
  >
    <h1
      className="page__title"
    >
      Aaaah! Something went wrong.
    </h1>
    <div
      className="page_body"
    >
      <p>
        Maybe the page or item you were looking for doesn't exist or something else is broken.
      </p>
      <p>
        Sorry :(
      </p>
    </div>
    <div
      className="page__meta"
    >
      <img
        alt="Ron Swanson Throwing Away His Computer"
        className="error-image"
        src="/images/badbad.gif"
        title="Ron Swanson Throwing Away His Computer"
      />
      <Link
        className="button"
        replace={false}
        to="/"
      >
        Go home
      </Link>
    </div>
  </div>
</div>
`;
